checker/jdk/determinism/src/java/util/List.java

```diff
@@ -562,7 +562,7 @@ default void sort(@PolyDet List<E> this, @PolyDet("use") Comparator<? super E> c
      * @throws IndexOutOfBoundsException if the index is out of range
      *         (<tt>index &lt; 0 || index &gt;= size()</tt>)
      */
-    @PolyDet("up") E set(@PolyDet List<E> this, @PolyDet("use") int index, E element);
+    @PolyDet("up") E set(@PolyDet List<E> this, @PolyDet("up") int index, E element);
```

Comment:
`set` removes the element at `index`, so the signature should be the same as `remove`.